programs/fuzz/fuzz_pkcs7.c

```diff
@@ -0,0 +1,19 @@
+#include <stdint.h>
+#include "mbedtls/pkcs7.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
```

Comment:
Thanks for adding a fuzzer! This is something we'd like to expand, but haven't had time to work on. Fuzzing more code is good.